tests/Fixtures/regression/simple_xml_element.test

```diff
@@ -1,7 +1,7 @@
 --TEST--
 Twig is able to deal with SimpleXMLElement instances as variables
 --CONDITION--
-version_compare(phpversion(), '5.3.0', '>=')
+version_compare(phpversion(), '8.0', '<')
```

Comment:
Does not work anymore in PHP 8.0. I won't investigate this further as this is an edge case anyway and not sure if we will be able to fix it "easily".